includes/admin-templates/new-template.php

```diff
@@ -97,3 +98,5 @@
 		<button id="elementor-new-template__form__submit" class="elementor-button elementor-button-success"><?php echo esc_html__( 'Create Template', 'elementor' ); ?></button>
 	</form>
 </script>
+
```

Comment:
Remove empty lines

Reply:
Done
1 line is added upon saving the file